openshift/elastalert/lad-elastalert-deployment.yaml

```diff
@@ -0,0 +1,227 @@
+        writeback_index: ${METADATA_STORE_INDEX}
+
+        # If an alert fails for some reason, ElastAlert will retry
+        # sending the alert until this time period has elapsed
+        alert_time_limit:
+          days: 7
+
+        scan_subdirectories: false
+
+        smtp_host: ${SMTP_SERVER}
+
+  - apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: elastalert-rules
+    data:
+      lad-anomaly-detected.yaml: |
+        name: "LAD ANOMALIES DETECTED"
+
+        es_host: ${ELASTICSEARCH_HOST}
+        es_port: ${ELASTICSEARCH_PORT}
+        use_ssl: False
+        verify_certs: False
+        use_strftime_index: true
+        index: "odcs-anomaly-detect-%Y.%m.%d"
+        type: frequency
+        num_events: 1
+        timeframe:
+          minutes: 2
+
+        realert:
+          minutes: 10
+
+        filter:
+        - query:
+            query_string:
+              query: "anomaly:1"
+
+        alert_text: |
+            We found the following message in your logs:
+            message: {2}
+
+            If this is not correct and you would like to report it you can provide feedback with the following link:
+            ${FACTSTORE_URL}?lad_id={0}&is_anomaly=False&message={1}
+
+        alert_text_args: ["predict_id", "e_message", "message"]
+        alert_text_type: alert_text_only
+
+        alert:
+        - "email"
+        email:
+        - "${ALERTS_EMAIL}"
+
+      lad-anomaly-no-logs.yaml: |
+        name: "LAD ANOMALY DETECT logs are not seen for an hour"
+
+        es_host: ${ELASTICSEARCH_HOST}
+        es_port: ${ELASTICSEARCH_PORT}
+        use_ssl: False
+        verify_certs: False
+
+        use_strftime_index: true
+        index: "lad-anomaly-detect-%Y.%m.%d"
+
+        use_count_query: true
+        doc_type: log
+        type: flatline
+
+        threshold: 1
+        timeframe:
+          minutes: 60
+        realert:
+          hours: 1
+
+        alert:
+        - "email"
+        email:
+        - "${ALERTS_EMAIL}"
+
+parameters:
+  - name: ELASTICSEARCH_HOST
+    displayName: Host URL for elasticsearch
+    value: "lad-elasticsearch-service.lad.svc"  # assuming lad is the namespace
+    required: true
+  - name: ELASTICSEARCH_PORT
+    displayName: Host port for elasticsearch
+    value: "9200"
+    required: true
+  - name: ELASTICSEARCH_USER
+    displayName: username for elasticsearch connection
+    value: elastalert
+    required: true
+  - name: ELASTICSEARCH_PASSWORD
+    displayName: password for elasticsearch connection
+    value: elastalert
+    required: true
+  - name: FACTSTORE_URL
+    displayName: "Host URL for Factstore ending with a '/'"
+    required: true
+  - name: SMTP_SERVER
+    displayName: Host URL for smtp email server
+    required: true
+  - name: METADATA_STORE_INDEX
+    displayName: Host URL for Factstore
+    value: elastalert_status
+    required: true
+  - name: ALERTS_EMAIL
+    displayName: email address to send alerts to
+    value: zhassan@redhat.com
```

Comment:
@4n4nd Thanks for including my email here. Will look forward to spam emails. Thank you. Thank you very much...